### DISCUSSION TITLE
Reset settings after test in MacOS

### DISCUSSION THREAD

**Discussion by RobinNagpal** [SafeExamBrowser/seb-mac#112] [General]
I am configuring a simple test with a URL and with "Preferences Menu Disabled". 

Now when I try to take that test and after I exit, the settings of the test are retained and I am not able to open the preferences even when I directly(not by clicking the .seb file) open SafeExamBrowser.

The only way to reset the settings now is to delete the ~/Library/Preferences/<SEB plist file>. Is there any way in which the settings can be reset after exiting the exam?

**Comment by danschlet** [comment]
New SEB versions (2.3 or later) require a changed command in Terminal:

`defaults delete org.safeexambrowser.SafeExamBrowser `(for SEB 2.3 and newer) or `defaults delete org.safeexambrowser.Safe-Exam-Browser` (SEB versions before SEB 2.3).

**Comment by ghost** [comment]
When I use this command the next time I open SEB I get a warning screen 
![IMG_0575](https://github.com/user-attachments/assets/fef20ff4-d0a1-44dd-a13b-2ec90559c108)

**Reply by danschlet** [reply]
This is correct. SEB let's you know that settings have been reset.

**Reply by SquirmDog** [reply]
The trouble is the settings option is gray'ed out and I can't modify any of my settings files. This is after removing the .plist setting (defaults delete org.safeexambrowser.SafeExamBrowser). Using macOS 15.7.4 and SEB 3.6.1

<img width="1441" height="732" alt="Screenshot 2026-03-31 at 6 49 45 AM" src="https://github.com/user-attachments/assets/ef2162cb-ff9d-4236-b424-67b2b3f84b9a" />

[Edit]: If SEB is configured with _"Allow to open Settings window on client"_ **unchecked**, then the settings menu option is grayed out and the command+, shortcut disabled. Deleting the client settings in located in `~/Library/Preferences/org.safeexambrowser.SafeExamBrowser.plist` does not revert SEB to a "fresh install" state and you are still locked out of settings. Creating a new config on another device that configures the client with allow settings checked corrects the problem. I'm not sure if creating a new user on macOS would do the same as I'm not sure where the SEB setting actually resolves
